docs/content/en/blog/news/control-plane-on-ecs.md

```diff
@@ -0,0 +1,285 @@
+]
+entrypoint = [
+  "sh",
+  "-c"
+]
+secrets = [
+  {
+    "name" : "ENCRYPTION_KEY",
+    "valueFrom" : "arn:aws:secretsmanager:${data.aws_region.current.id}:${data.aws_caller_identity.self.account_id}:secret:${var.encryption_key_secret}"
+  },
+  {
+    "name" : "CONTROL_PLANE_CONFIG",
+    "valueFrom" : "arn:aws:secretsmanager:${data.aws_region.current.id}:${data.aws_caller_identity.self.account_id}:secret:${var.control_plane_config_secret}"
+  },
+]
+},
+```
+
+### ALB
+You must prepare two target groups for both HTTP and gRPC. Make two hosts and listner rules as below. Listner protocol should be HTTPS becuase it uses gRPC.
+
+![](/images/control-plane-alb.png)
+
+### Terraform example
+PipeCD gives [control-plane-aws-ecs-terraform-demo](https://github.com/pipe-cd/control-plane-aws-ecs-terraform-demo)
+
+#### Prepare
+1. Prepare SSL certificate
+Prepare your domain and SSL certificate from AWS certificate manager.
+
+2. Create a s3 bucket for terraform backend
+Write bucket name to `00-main.tf`
+```
+terraform {
+  backend "s3" {
+    bucket  = "example-pipecd-control-plane-tfstate" #your bucket name for terraform backend
+    region  = "ap-northeast-1"
+    key     = "tfstate"
+    profile = "pipecd-control-planeg-terraform" #your profile
+  }
+  required_providers {
+    aws = {
+      version = "~> 3.34.0"
+    }
+  }
+}
+```
+
+3. Edit `variables.tf` for your project
+```
+//export
+locals {
+  alb = {
+    certificate_arn = ""
+  }
+  
+  redis = {
+    node_type = "cache.t2.micro"
+  }
+
+  rds = {
+    node_type = "db.t3.micro"
+  }
+
+  ecs = {
+    memory = "1024"
+    cpu = "512"
+  }
+}
+```
+
+4. Create a S3 bucket for filestore and write the bucket name for it to `control-plane-config.yaml` and `variables.tf`
+```
+apiVersion: "pipecd.dev/v1beta1"
+kind: ControlPlane
+spec:
+    datastore:
+    type: MYSQL
+    config:
+        url: root:test@tcp(pipecd-mysql:3306)
+        database: quickstart
+    filestore:
+    type: S3
+    config: # edit here
+        bucket: example-pipecd-control-plane-filestore 
+        region: ap-northeast-1
+    projects:
+    - id: quickstart
+        staticAdmin:
+        username: hello-pipecd
+        passwordHash: "$2a$10$ye96mUqUqTnjUqgwQJbJzel/LJibRhUnmzyypACkvrTSnQpVFZ7qK" # bcrypt value of "hello-pipecd"
+```
+```
+//export
+locals {
+  s3 = { # These must be unique in the world.
+    filestore_bucket = "${local.project}-filestore" # edit here
+  }
+}
+```
+5. Write config of RDS for datastore to `control-plane-config.yaml`
+Note: Do not edit hostname (pipecd-mysql) because it will be edited autimaticaly by terraform.
+```
+apiVersion: "pipecd.dev/v1beta1"
+kind: ControlPlane
+spec:
+    datastore:
+    type: MYSQL
+    config: # edit here
+        url: root:test@tcp(pipecd-mysql:3306)
+        database: quickstart
+    filestore:
+    type: S3
+    config: 
+        bucket: example-pipecd-control-plane-filestore 
```

Comment:
Does Terraform create this bucket in the example repo?

Reply:
No, users must make a bucket and edit config file according to user's bucket.